pymedphys/labs/quickcheck/qcheck.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+"""
+Class based QUICKCHECK connection
+
+@author: Rafael Ayala
+"""
+import codecs
+import datetime
+import re
+import socket
+
+from tqdm import tqdm
```

Comment:
There's a set up within PyMedPhys where all non-std-lib imports are made to be optional. It is set up so that pymedphys can be run without those installed, and they're not imported until they actually need to be used.

To support this you actually need to run:

```python
from pymedphys._imports import tqdm
```

Then when you call it below use `tqdm.tqdm`